authn.proto

```diff
@@ -39,11 +39,13 @@ message Token {
     string value = 1;
 }
 
-message UserID {
-    string value = 1;
+message UserIdentity {
```

Comment:
Maybe just `User`? Like - do we know what else in the future might be sent.

Btw. do we still need e-mail for AuthN at all?

Reply:
I prefer to keep `UserIdentity` because it does not represent a User, but a response from the Identify method. I think that we can remove an email, but that requires some modifications in the Users service and it's probably better to do it in a separate PR.

Reply:
I created an issue for this: #1273.